scanner/cmd/scannerctl/main.go

```diff
@@ -41,14 +41,10 @@ func rootCmd(ctx context.Context) *cobra.Command {
 	}
 	cmd.SetContext(ctx)
 	flags := cmd.PersistentFlags()
-	indexerAddr := flags.String(
-		"indexer-address",
-		"",
-		"Address of the indexer service.")
-	matcherAddr := flags.String(
-		"matcher-address",
-		"",
-		"Address of the matcher service.")
+
+	cmd.PersistentFlags().String("indexer-address", "", "Address of the indexer service")
```

Comment:
just curious why make these changes?

Reply:
To [read the flag](https://github.com/stackrox/stackrox/pull/15991/files#diff-0d62184f16cb459df5e721ae98e804bc0aaca29020ad8981802118aea32707dfR188).

Reply:
The link doesn't seem to be taking me anywhere for some reason, but I think it's so we can use it in scale.go?